tests/end2end/data/scroll/frame.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Simple HTML inside an iframe</title>
+    </head>
+    <body>
+        <iframe id="scroll-frame" style="margin: 50px;" src="/data/scroll/simple.html"></iframe>
+
+    </body>
+
+    <script>
```

Comment:
put the script tag inside the body, the html tag may only have head and body as children.

Reply:
Good catch! Might even be my mistake in some file @jgkamat copied this from :blush:

Reply:
Nah, I think this is just my mistake... :P 

I think that there might be a lot of other errors/warnings, we might want to set up CI for html validation, but I don't think that would actually help qb too much :P